lib/api/core/auth/passwordManager.js

```diff
@@ -0,0 +1,21 @@
+var crypto = require('crypto');
+
+module.exports = function PassportManager () {
+
+  this.secret = 'jbsB8r69PFP39KdLtjVr25Z22';
+  this.algo = 'sha1';
+  this.digest = 'hex';
+
+  this.encryptPassword = function(password) {
+    var hashedPassword = crypto.createHmac(this.algo, this.secret).update(password).digest(this.digest);
+    return Promise.resolve(hashedPassword);
```

Comment:
should return ``q(hashedPassword)`` instead of ``Promise.resolve`` to ensure promises compatibility.